### PR TITLE
TS-19-74 custom sidebar

### PR DESCRIPTION
**Pull request by Niknikson**

### CODE REVIEW

File: frontend/src/share/SearchPanel/SearchPanel.scss [hunk 1]
```diff
@@ -1,15 +1,31 @@
...
-
-    @media (max-width: 500px) {
-        width: 85% !important;
+    padding: 20px 20px 30px;
+    border-radius: 2px;
+    margin: 10px;
+    border: 1px solid $light-gray;
+    -webkit-box-shadow: 4px 4px 8px -4px rgba(34, 60, 80, 0.2);
+    -moz-box-shadow: 4px 4px 8px -4px rgba(34, 60, 80, 0.2);
+    box-shadow: 4px 4px 8px -4px rgba(34, 60, 80, 0.2);
```
Comment (ViktoriiaKosovan) [inline frontend/src/share/SearchPanel/SearchPanel.scss]: Are you sure that we need this vendor prefixes? create-react-app supports the vendor prefixes based on the configuration.

**Review by ViktoriiaKosovan — APPROVED** [review]

File: frontend/src/share/SearchPanel/SearchPanel.scss (hunk 1 above)
Comment (HiroFive) [reply frontend/src/share/SearchPanel/SearchPanel.scss]: And `rgba(34, 60, 80, 0.2);` to constants or use a ready-made constant `box-shadow`

File: frontend/src/share/SearchPanel/SearchPanel.scss [hunk 2]
```diff
@@ -1,15 +1,29 @@
...
+    margin: 10px;
+    border: 1px solid $light-gray;
+    box-shadow: 4px 4px 8px -4px $box-shadow;
+    .MuiTypography-body1{
+        font-size: 0.8rem;
     }
+
 }
 
+ @media (max-width: 850px) {
```
Comment (HiroFive) [inline frontend/src/share/SearchPanel/SearchPanel.scss:16]: create constant for this media

File: frontend/src/styles/colors.scss [hunk 3]
```diff
@@ -9,21 +9,21 @@ $shadow-box: #00000042;
...
+$light-blue: #add8e6;
+$dark-blue: #3f51b5;
 $blue: #1976d2;
 $blue-hover: #155da5;
-$dark-blue: #3f51b5;
-$selected-blue: #0000001a;
 $light-forest: #028802;
 $forest: #184b44;
+
+$box-shadow: rgba(34, 60, 80, 0.2);
```
Comment (HiroFive) [inline frontend/src/styles/colors.scss:29]: use hex format

File: frontend/src/share/SearchPanel/SearchPanel.scss [hunk 4]
```diff
@@ -1,15 +1,29 @@
...
+     .search-group{
+        display: flex;
+        align-items: center;
+        flex-direction: column;
+     }   
+    }
+
 .form-field {
     @media (max-width: 800px) {
         max-width: 100% !important;
```
Comment (HiroFive) [inline frontend/src/share/SearchPanel/SearchPanel.scss:26]: don't use `!important`

File: frontend/src/share/SearchPanel/SearchPanel.scss [hunk 5]
```diff
@@ -1,15 +1,29 @@
...
-
-    @media (max-width: 500px) {
-        width: 85% !important;
+    padding: 20px 20px 30px;
+    border-radius: 2px;
+    margin: 10px;
+    border: 1px solid $light-gray;
+    box-shadow: 4px 4px 8px -4px $box-shadow;
+    .MuiTypography-body1{
+        font-size: 0.8rem;
```
Comment (HiroFive) [inline frontend/src/share/SearchPanel/SearchPanel.scss:11]: use for this font-size: `font-xs`

**Pull request closed without merging**